CorsixTH/Src/persist_lua.h

```diff
@@ -146,27 +131,24 @@ class lua_persist_reader
     }
 
     template <class T>
-    bool read_int(T& tValue)
-    {
+    bool read_int(T& tValue) {
         typename lua_persist_int<T>::T tWrittenValue;
-        if(!read_uint(tWrittenValue))
-            return false;
-        if(tWrittenValue & 1)
+        if (!read_uint(tWrittenValue)) return false;
+        if (tWrittenValue & 1)
             tValue = (-(T)(tWrittenValue >> 1)) - 1;
         else
             tValue = static_cast<T>(tWrittenValue >> 1);
         return true;
     }
 
     template <class T>
-    bool read_float(T& fValue)
-    {
-        if(!read_byte_stream(reinterpret_cast<uint8_t*>(&fValue), sizeof(T)))
+    bool read_float(T& fValue) {
+        if (!read_byte_stream(reinterpret_cast<uint8_t*>(&fValue), sizeof(T)))
```

Comment:
Code can be shorter by returning the value of the function call, but for formatting, does it make sense to have multi-line statements without curly brackets?

Reply:
No, the new style requires braces here; just wasn't automatically picked up and I missed it.

Reply:
It's only a single line - Google style allows it.